### PR TITLE
feat: Redirect "not enough mana", healing and speed effect messages

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

**Comment by kristofbolyai** [comment]
No background message regex because spells cannot be cast when in dialouge.

**Comment by kristofbolyai** [comment]
> No background message regex because spells cannot be cast when in dialouge.

Even if there is a way it would appear, it is not worth to filter for it.

File: common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java [hunk 1]
```diff
@@ -106,6 +108,9 @@ public class InfoMessageFilterFeature extends UserFeature {
     @Config
     private FilterType craftedDurability = FilterType.REDIRECT;
 
+    @Config
+    private FilterType notEnoughMana = FilterType.REDIRECT;
+
     @SubscribeEvent
     public void onInfoMessage(ChatMessageReceivedEvent e) {
         if (e.getRecipientType() != RecipientType.INFO) return;
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java:130]: Not strictly this pr, but we need to filter dialogue messages from this check and spamming queue

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java:130]: There is already an issue for it

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java:130]: https://github.com/Wynntils/Artemis/issues/261

**Review by P0keDev — APPROVED** [review]

**Comment by magicus** [comment]
>    >No background message regex because spells cannot be cast when in dialouge.
>
> Even if there is a way it would appear, it is not worth to filter for it.

Just to clarify: of course spells can be cast when in dialogue. You can do whatever you want, it's just that Wynncraft will keep resending chat history until the NPC has finished talking. But your second assessment might still apply, that it is not worth it to fix.

**Comment by kristofbolyai** [comment]
> > > No background message regex because spells cannot be cast when in dialouge.
> > 
> > 
> > Even if there is a way it would appear, it is not worth to filter for it.
> 
> Just to clarify: of course spells can be cast when in dialogue. You can do whatever you want, it's just that Wynncraft will keep resending chat history until the NPC has finished talking. But your second assessment might still apply, that it is not worth it to fix.

No, it did not seem to me like that I can cast anything. Weird...

**Comment by magicus** [comment]
What? Is that so? I need to check that :)

edit: You are absolutely correct. Funny I never realized that.

**Pull request merged**